content/docs/faq-versioning.md

```diff
@@ -1,48 +1,52 @@
 ---
 id: faq-versioning
-title: Versioning Policy
+title: سياسة الإصدارات
 permalink: docs/faq-versioning.html
 layout: docs
 category: FAQ
 ---
 
-React follows [semantic versioning (semver)](https://semver.org/) principles.
+تتبع React مبادئ [الإدارة الدلالية لنُسخ البرمجيات (SemVer)](https://semver.org/) في ترقيم إصداراتها.
 
-That means that with a version number **x.y.z**:
+هذا يعني أنَّه مع الإصدار ذي الرقم **x.y.z**:
 
-* When releasing **breaking changes**, we make a **major release** by changing the **x** number (ex: 15.6.2 to 16.0.0).
-* When releasing **new features**, we make a **minor release** by changing the **y** number (ex: 15.6.2 to 15.7.0).
-* When releasing **bug fixes**, we make a **patch release** by changing the **z** number (ex: 15.6.2 to 15.6.3).
+* عند إصدار **تغييرات جذرية** (breaking changes)، نطلق **إصدارًا رئيسيًّا** عبر تغيير الرقم **x** (مثل الانتقال من 15.6.2 إلى 16.0.0).
+* عند إصدار **ميزات جديدة**، نطلق **إصدارًا فرعيًّا** عبر تغيير الرقم **y** (مثل التغيير من 15.6.2 إلى 15.7.0).
+* عند **تصحيح علل وثغرات** في الإصدار السابق، نطلق **إصدارًا مرقَّعًا** (patch release) عبر تغيير الرقم **z** (مثل تغيير 15.6.2 إلى 15.6.3).
 
-Major releases can also contain new features, and any release can include bug fixes.
+قد تحوي الإصدارات الرئيسية على ميزات جديدة أيضًا، ويمكن أي يحوي أي إصدار على تصحيح لعللٍ وثغرات.
 
-### Breaking Changes {#breaking-changes}
 
```

Comment:
Hi @Masked Developer, thank you very much for you're effort 😄 !
Please, I suggested some small fixes and a request to change an expression. Can you check it out?!